itou/utils/apis/metabase.py

```diff
@@ -9,6 +9,7 @@
 ASP_SIAE_FILTER_KEY_FLAVOR2 = "id_asp_de_la_siae"
 ASP_SIAE_FILTER_KEY_FLAVOR3 = "id_asp_siae"
 C1_SIAE_FILTER_KEY = "identifiant_de_la_structure_(c1)"
+C1_ORG_FILTER_KEY = "id_prescripteur"
```

Comment:
Coté C1 le terme "organisation" sert pour toutes les structures donc je préciserais dans le nom de la constante
```suggestion
C1_PRESCRIBER_ORG_FILTER_KEY = "id_prescripteur"
```